ctms/crud.py

```diff
@@ -79,25 +79,45 @@ def _contact_base_query(db):
     )
 
 
+def query_filter_list(start_time, end_time, after_email_uuid, mofo_relevant):
```

Comment:
Nit: I'd do this work ``get_bulk_contents``, since it doesn't seem like a generally useful function.

Reply:
renamed this